app/assets/javascripts/short-form/ShortFormDataService.js.coffee

```diff
@@ -447,7 +449,7 @@ ShortFormDataService = (ListingDataService, ListingConstantsService, ListingPref
       return unless listingPref
 
       member = _.find(allHousehold, {appMemberId: shortFormPref.appMemberID})
-
+      isCustom = false
```

Comment:
I know it's less efficient but it might be more readable to to

```
isCustom = !Object.values(ListingDataService.preferenceMap).includes(listingPref.preferenceName)
```

and then you can get rid of line 473

Reply:
I kind of like having the whether or not it's a custom preference just be in one place here because the if/else statements are already really confusing 😕. I think the way I put it is actually less confusing to me?? Idk all of the if elses are just pretty crazy

Reply:
okay, fine to keep it then